src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Tasks/KeyToolTests.cs

```diff
@@ -163,5 +165,47 @@ public void CreateDebugKeyStore ()
 
 			Assert.IsTrue (keyToolTask.Execute (), "Task should have succeeded.");
 		}
+
+		[Test]
+		public void CreateDebugKeyStoreWithStrongPassword ()
+		{
+			string keyfile = Path.Combine (TestName, "debug.keystore");
+			string pass = "Cy(nBW~j.&@B-!R_aq7^/syzFR!S$4]7R%i6)R!";
+			if (File.Exists (keyfile))
+				File.Delete (keyfile);
+			var task = new AndroidCreateDebugKey {
+				BuildEngine = engine,
+				KeyStore = keyfile,
+				StorePass = pass,
+				KeyAlias = "teststringkey",
+				KeyPass = pass,
+				KeyAlgorithm="RSA",
+				Validity=10000,
+				StoreType="pkcs12",
+				Command="-genkeypair",
+				ToolPath = keyToolPath,
+			};
+			Assert.IsTrue (task.Execute (), "Task should have succeeded.");
+			Assert.AreEqual (0, errors.Count, "Task should have no errors.");
+			Assert.AreEqual (0, warnings.Count, "Task should have no warnings.");
+			Assert.AreEqual (0, task.ExitCode, "ExitCode should have been 0");
+
+			messages.Clear ();
+			var keyToolTask = new KeyTool {
+				BuildEngine = engine,
+				KeyStore = keyfile,
+				StorePass = pass,
+				KeyAlias = "teststringkey",
+				KeyPass = pass,
+				Command = "-list",
+				ToolPath = keyToolPath,
+			};
+
+			Assert.IsTrue (keyToolTask.Execute (), "Task should have succeeded.");
+			Assert.AreEqual (0, errors.Count, "Task should have no errors.");
+			Assert.AreEqual (0, warnings.Count, "Task should have no warnings.");
+			Assert.AreEqual (0, task.ExitCode, "ExitCode should have been 0");
+			Assert.IsTrue (messages.Select (x => x.Message).Contains ("Certificate fingerprint (SHA1):"), "Certificate SHA1 should have been printed.");
```

Comment:
This assert is failing: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3640507&view=ms.vss-test-web.build-test-results-tab&runId=12594914&resultId=100127&paneView=debug

```
Certificate SHA1 should have been printed.
 Expected: True
 But was: False
```

Reply:
Weird cos its in the output `Certificate fingerprint (SHA1):` ... I'll take a look